src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -489,8 +493,40 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, correlat
                                disable_auto_upgrade, enable_custom_locations, custom_locations_oid,
                                helm_client_location, enable_private_link, arm_metadata,
                                onboarding_timeout, container_log_path)
+    
+    # Add 2nd long running operation to wait for Agent State to reach terminal stage with a default 20 minute timeout window 
+    # TODO: Update poll_for_agent_state method to check for agent state and not provisioning state when feedback loop is implemented
+    if enable_oidc_issuer or enable_workload_identity:
```

Comment:
if only enable_oidc_issuer is issued by the end user,will it be usefull to hold for the connected state?

Reply:
Why would we have different functionality if only oidc is enabled vs if both are enabled?